tests/integration/test_filters.py

```diff
@@ -243,6 +243,7 @@ def test_ignore_transactions_filters_are_applied(
     project_id = 42
     project_config = mini_sentry.add_full_project_config(project_id)
     filter_settings = project_config["config"]["filterSettings"]
+    del project_config["config"]["transactionMetrics"]
```

Comment:
not adding the transactionMetrics config to the basic_project_config made a lot of tests fail, adding it made another set of tests fail, manually deleting it for the latter set is what ended up working.  it feels a bit strange way to do it but let me know your thoughts please

Reply:
I would prefer additive construction: remove this field from the `basic_project_config`, remove all the `del` statements, and then use a helper function `make_config` that adds the `transactionMetrics` config in the tests that need it.